terraform/ec2/localstack/main.tf

```diff
@@ -57,6 +57,9 @@ resource "aws_instance" "integration-test" {
         "aws s3 cp s3://${var.s3_bucket}/integration-test/cloudwatch-agent-test-repo/${var.cwa_github_sha}.tar.gz ./amazon-cloudwatch-agent-test.tar.gz --quiet",
         "mkdir amazon-cloudwatch-agent-test",
         "tar -xzf amazon-cloudwatch-agent-test.tar.gz -C amazon-cloudwatch-agent-test",
+        "echo Downloading LocalStack image from S3...",
+        "aws s3 cp s3://${var.s3_bucket}/integration-test/docker-images/${var.cwa_github_sha}.tar . --quiet",
```

Comment:
Don't need to have a separate one for each GitHub SHA. It's not tied to that and seems like a waste of storage to keep a copy of each.